prepare_pacta_indices.R

```diff
@@ -6,9 +6,6 @@ suppressPackageStartupMessages({
   library("readr")
 })
 
-# config
-readRenviron(".env")
```

Comment:
doesn't work for me without this...
``` r
writeLines(c("PACTA_DATA_PATH=../pacta-data/", "R_CONFIG_ACTIVE=2022Q4"),  ".env")

Sys.getenv("R_CONFIG_ACTIVE")
#> [1] ""

config <- config::get(
  file = "config.yml",
  config = Sys.getenv("R_CONFIG_ACTIVE"),
  use_parent = FALSE
)

config$pacta_financial_timestamp
#> [1] "2021Q4"
```

Reply:
suggestion
```r
if (interactive() && Sys.getenv("R_CONFIG_ACTIVE") == "" && file.exists(".env")) {
  readRenviron(".env")
}
```

Reply:
> This file can be replaced or overridden by specifying environment variables on the host machine, or as part of invoking docker.

From the changes in the `### Running in docker-compose` section of README. 

Would you prefer if we had
```r
if(interactive()){
  readRenviron(".env")
}
```

Reply:
yes-ish, but.... experimenting further with the script, it's impossible to run in RStudio anyway because it relies on the `docker-compose.yml` to mount directories that are hard-coded, so I think it probably doesn't matter